Update maistra GA version

mkralik3 opened the pull request.

FilipB commented:
Blocked by https://github.com/openshift/release/pull/50251

FilipB commented:
/ok-to-test

FilipB commented:
/pj-rehearse

FilipB commented:
/retest

FilipB commented:
/assign @FilipB

mkralik3 commented:
/pj-rehearse

FilipB commented:
/pj-rehearse max

mkralik3 commented:
do not merge.
it is waiting for https://github.com/maistra/test-infra/pull/391 to check if permissions are correct ( also `tmp` commit needs to be reverted)

mkralik3 commented:
/pj-rehearse pull-ci-maistra-maistra-test-tool-main-build

mkralik3 commented:
/retest pull-ci-maistra-maistra-test-tool-main-build

mkralik3 commented:
/pj-rehearse max

mkralik3 commented:
/pj-rehearse pull-ci-maistra-maistra-test-tool-main-build

mkralik3 commented:
/pj-rehearse pull-ci-maistra-maistra-test-tool-main-build

mkralik3 commented:
/pj-rehearse pull-ci-maistra-maistra-test-tool-main-build

mkralik3 commented:
/pj-rehearse max

mkralik3 commented:
/retest-required

mkralik3 commented:
/retest-required

Sau1506mya commented:
Hi @mkralik3 , please run `make update` to update the periodic jobs as well. Otherwise `lgtm`.

mkralik3 commented:
@Sau1506mya Hello. I have run `make update` in my localhost. However, nothing was changed in files. Did it update something remotely?

Sau1506mya commented:
/lgtm

mkralik3 commented:
@Sau1506mya can I somehow trigger those periodic jobs here to test if my changes don't break anything there? ( it looks like only rehearse jobs were triggered )

Sau1506mya commented:
> @Sau1506mya can I somehow trigger those periodic jobs here to test if my changes don't break anything there? ( it looks like only rehearse jobs were triggered )

Hi @mkralik3 , Which periodic job you want to re-trigger?
Btw, periodic job can be triggered by `/pj-rehearse job_name1 job_name2 ...` and so on .

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

Sau1506mya commented:
/lgtm cancel

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-interop-servicemesh-interop-aws

mkralik3 commented:
MTT and kiali int tests passed
There are some flaky tests in Kiali Cypress v1.73 test container ( and some failures ) .. I am going to investigate.

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

Sau1506mya commented:
Hi @mkralik3 , Can you also re-trigger the job for other periodic jobs, they failed previously so required them also to be passed?

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.15-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.13-lp-interop-servicemesh-interop-aws

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.15-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.13-lp-interop-servicemesh-interop-aws

mkralik3 commented:
/pj-rehearse max

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.13-lp-interop-servicemesh-interop-aws

mkralik3 commented:
@Sau1506mya  Hello, only `aws-rosa-hypershift` jobs remain failing. However, it looks like a problem with provisioning clusters. So PR is ready to merge.

Sau1506mya commented:
> @Sau1506mya Hello, only `aws-rosa-hypershift` jobs remain failing. However, it looks like a problem with provisioning clusters. So PR is ready to merge.

Hi @mkralik3 , Is hypershift job not active for servicemesh: `periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift`? 
I mean is the servicemesh job not being in weekly testing for ROSA STS Hypershift Layered Product scenarios?

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

mkralik3 commented:
@Sau1506mya  I don't know.
According to history, it is. https://prow.ci.openshift.org/job-history/gs/origin-ci-test/logs/periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift 

Let me rerun the job again ( it failed on some cluster related steps )

Btw, I didn't see a history for `hypershift-4.13-lp-rosa` and `hypershift-4.12-lp-rosa` jobs. Can they be removed if they are not running?

mkralik3 commented:
@Sau1506mya   can I remove the old ones then? (4.12 , 4.13, 4.14)

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

mkralik3 commented:
/pj-rehearse ci/prow/ci-operator-registry

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.15-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.13-lp-interop-servicemesh-interop-aws

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.15-lp-interop-servicemesh-interop-aws

mkralik3 commented:
/pj-rehearse ack

mkralik3 commented:
@Sau1506mya  I have added `periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift` job and it passed. So PR is ready.

What should I do with those old hypershift-rosa jobs which don't work? (4.12-4.14) Should I delete them or disable them? If yes, how?

Sau1506mya commented:
/test ci-operator-registry

Sau1506mya commented:
Can we retest the ROSA Classic job :`periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-rosa-classic-servicemesh-rosa-aws`?

mkralik3 commented:
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-rosa-classic-servicemesh-rosa-aws

Sau1506mya commented:
So @mkralik3 , we are good to go. All tests are passing in all the platforms including interop, ROSA Classic and Hypershift.

Sau1506mya commented:
/pj-rehearse ack

mkralik3 commented on ci-operator/jobs/maistra/maistra-test-tool/maistra-maistra-test-tool-main-periodics.yaml:
```diff
@@ -253,6 +253,87 @@ periodics:
+    ci.openshift.io/generator: prowgen
+    job-release: "4.15"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift
```
@calebevans thx, I will update all jobs with that.

Sau1506mya commented:
/hold

Sau1506mya commented:
/hold cancel

Sau1506mya commented:
Hi @mkralik3 , the PR is good to merge but for this all checks need to be passed. It's still in the waiting stage. Can you please try to rebase the PR or make some changes such that it will come up in the stack and then may be it will run the checks. I tried with `hold` but, it didn't worked.

Sau1506mya commented:
> Hi @mkralik3 , the PR is good to merge but for this all checks need to be passed. It's still in the waiting stage. Can you please try to rebase the PR or make some changes such that it will come up in the stack and then may be it will run the checks. I tried with `hold` but, it didn't worked.

You can try with adding the slack config to these jobs:
`periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.13-lp-interop-servicemesh-interop-aws`
`periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.13-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift`
`periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.12-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift-412 `
In this way, a change will be made in the PR and hopefully it will run the checks.

mkralik3 commented:
@Sau1506mya  I have rebased it with master branch and added slack reporter config there. Should I run pj-rehearse again? ( no changes related to jobs were done )

Sau1506mya commented:
/pj-rehearse ack

calebevans reviewed (approved):
/lgtm

FilipB commented:
/approve
